Vagrantfile

```diff
@@ -34,12 +34,20 @@ Vagrant.configure(2) do |config|
     controlplane.vm.box = "roboxes/ubuntu2004"
 
     controlplane.vm.network :private_network, ip:"#{k8s_linux_kubelet_nodeip}"
-    controlplane.vm.provider :virtualbox do |vb|
-    controlplane.vm.synced_folder "./sync/shared", "/var/sync/shared"
-    controlplane.vm.synced_folder "./forked", "/var/sync/forked"
-    controlplane.vm.synced_folder "./sync/linux", "/var/sync/linux"
-      vb.memory = linux_ram
-      vb.cpus = linux_cpus
+    controlplane.vm.synced_folder ".", "/vagrant", disabled: true
+    controlplane.vm.synced_folder "./sync/shared", "/var/sync/shared", type: "rsync"
+    controlplane.vm.synced_folder "./forked", "/var/sync/forked", type: "rsync"
+    controlplane.vm.synced_folder "./sync/linux", "/var/sync/linux", type: "rsync"
+
+    controlplane.vm.provider "qemu" do |qe|
+      qe.memory = linux_ram
+      qe.qemu_dir = "/opt/homebrew/share/qemu"
```

Comment:
This directory is going to be different based on the base OS. Does it have a sane default based on the OS? IOW, do we need to specify it here?

Reply:
I remove and tested the VM bringup without this statement. And it works. So, should be ok to remove it.